.changeset/itchy-seas-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fix `function-no-unknown` false positives for `scroll`, `-webkit-gradient`, `color-stop`, `from`, and `to`
```

Comment:
[suggest] Can you please change `Fix` to `Fixed:` (the common prefix in `CHANGELOG.md`)?

```suggestion
Fixed: `function-no-unknown` false positives for `scroll`, `-webkit-gradient`, `color-stop`, `from`, and `to`
```

https://github.com/stylelint/stylelint/blob/e30ec863f8bc1d5ed87c23e3c5a554a01bc4350b/CHANGELOG.md?plain=1#L7

Reply:
Will do. I just thought you wanted that when you said "I've updated the PR title for a changeset.".

Reply:
Oh, sorry. 🙇🏼